src/SFA.DAS.ToolsNotifications.Core/Configuration/Constants.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace SFA.DAS.ToolsNotifications.Core.Configuration
+{
+    public static class Constants
+    {
+        public const string ApiName = "Tool Service Notification API";
+        public const string PathBase = "/api/notifications";
+        public const string RedisKey = "das-tools-notification";
+        public const string AuthorizationPolicyName = "RequireNotificationRole";
+        public const string AuthorizationRequiredRoleName = "Notifications";
+        public const string ScopeClaimType = "http://schemas.microsoft.com/identity/claims/scope";
+        public const string ObjectIdClaimType = "http://schemas.microsoft.com/identity/claims/objectidentifier";
```

Comment:
I don't think any of this belongs in Core, it either relates to the frontend or backend and not core business logic

Reply:
I put this in core because these settings could be shared between projects.. e.g `RedisKey` is currently referenced in `NotificationRedisRepository`.